app/kumactl/cmd/install/testdata/install-control-plane.cni-experimental-enabled.golden.yaml

```diff
@@ -2035,7 +2035,7 @@ spec:
       terminationGracePeriodSeconds: 5
       containers:
         - name: install-cni
-          image: "docker.io/kumahq/kuma-cni:1.7.0"
+          image: "docker.io/kumahq/kuma-cni:0.0.1"
```

Comment:
Where is `0.0.1` coming from? :thinking:

Reply:
https://github.com/kumahq/kuma/blob/f15239f7c13d0274b2387410f2bd8dd984327670/app/kumactl/cmd/install/install_control_plane_test.go#L31

Reply:
ah nice, ok